### ISSUE TITLE
Keys generated without email still show "<>" after name

### ISSUE THREAD

**Issue by gulkily** [openpgpjs/openpgpjs#735]
I am generating PGP keys that do not have an email address. When I do this in command-line GPG and then display the user ID with GPG, it is displayed as expected.

When I generate the key with OpenPGP.js, GPG displays it as "John Smith <>".

Any idea why this is happening or how to remedy it?

**Comment by twiss** [comment]
Could you post the version of OpenPGP.js and the code you're using?

Using the latest version, running

```js
await openpgp.generateKey({ userIds: [{ name: 'John Smith' }] })
```

and viewing that with GPG works fine for me.

**Comment by gulkily** [comment]
I'm using OpenPGP 3.0.3. Here are the relevant bits of the code:

```
            var openpgp = window.openpgp;
            openpgp.initWorker({path:'openpgp.worker.js'});

            var options = {
                userIds: [{ name: username }],
                numBits: 4096,
                passphrase: password
            };

            openpgp.generateKey(options).then(
                function(key) {
                }
            );

```

**Comment by twiss** [comment]
Yeah, I can reproduce the issue in 3.0.3. It has been fixed in the latest version, though. Please upgrade your version of OpenPGP.js if you can :)

**Comment by gulkily** [comment]
Confirmed fixed in 3.0.13. Thank you very much for your help!
